week2/index.js

```diff
@@ -0,0 +1,163 @@
+"use-strict";
+
+const fs = require('fs');
+const TODO_FILE = process.env['TODO_file'] || 'todo.json'
+const HELP_MENU = 'help.txt';

```

Comment:
This variable name is a bit weird, since it's not the help menu - it's just the filename of the help file. I would have gone with `HELP_FILENAME`, or something similar. Not `HELP_FILE`, because it's not the contents - just the filename.

Reply:
You could also do something like:
```
const FILENAMES = { help: "help.txt", todos: "todo.json" }
```
That way you just have a look-up table of all the filenames you might need, to be able to do `readFile( FILENAMES.help )`.